src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2019-to-cloud-2020.md

```diff
@@ -321,7 +321,66 @@ Workflow page providers now used the `ecm:isTrashed` attribute.
 
 ## Elasticsearch
 
-### Behavior Changes
+Nuxeo 11.4/Cloud 2021 is using Elasticsearch client library version 7.9.2.
```

Comment:
"Nuxeo 11.4/Cloud 2021" seems weird, not sure what terminology we want to use here: Nuxeo 11.4 and above / Nuxeo Cloud release, since 11.4 / ...

Reply:
ok will use Since Nuxeo 11.4, I guess there will be another review to fix the versionning for cloud/lts.